src/useWhatChanged.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 type TypeDependency = any[];
 type TypeDependencyNames = string;
 
 let what_debug_changed = 0;
+
+const [configuration, setConfiguration] = useState({ active: true });
```

Comment:
You are using useState outside any function component. Is it not failing. did you test it with example running the example folder.